components/rmrk/Gallery/GalleryCard.vue

```diff
@@ -111,6 +112,17 @@ export default class GalleryCard extends mixins(AuthMixin) {
   get accountIsCurrentOwner(): boolean {
     return this.accountId === this.currentOwner
   }
+
+  get layoutClass() {
+    if (
+      this.$store.getters['preferences/getLayoutClass'] ===
+      'is-one-quarter-desktop is-one-third-tablet'
+    ) {
+      return ''
+    }
+
+    return 'title is-4'
+  }
```

Comment:
What's this?

Reply:
oh, for this one, is it ok to adjust the text smaller in "Small Display" = 4 columns? if not, I will revert it

Reply:
then is suggest to use [vue style](https://v2.vuejs.org/v2/guide/class-and-style.html)

```html
<span
  :class="{ 'title is-4': smallDisplay  }"
></span

```


```ts
get smallDisplay(): boolean {
  this.$store.getters['preferences/getLayoutClass'] ===
      'is-one-quarter-desktop is-one-third-tablet'
}
```

Reply:
oh, nice, looks cleaner
will update that